### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by leostryuk**

### CODE REVIEW

File: dnq/src/main/kotlin/kotlinx/dnq/util/TransientEntityUtil.kt [hunk 1]
```diff
@@ -115,4 +116,10 @@ fun XdEntity.reattachAndGetLink(linkName: String): Entity? {
     return reattach(session).getLinkEx(linkName, session)
 }
 
-val XdEntity.isReadOnly: Boolean get() = (entity as TransientEntity).isReadonly
+val XdEntity.isReadOnly: Boolean
+    get() {
+        if (entity is OVertexEntity) {
+            throw IllegalStateException("LOshadinaya zalupa")
```
Comment (andrii0lomakin) [inline dnq/src/main/kotlin/kotlinx/dnq/util/TransientEntityUtil.kt]: Leonid I will kill you

**Review by andrii0lomakin — CHANGES_REQUESTED** [review]

**Comment by andrii0lomakin** [comment]
@leostryuk, please be more specific about changes.

File: dnq/src/main/kotlin/kotlinx/dnq/util/TransientEntityUtil.kt (hunk 1 above)
Comment (leostryuk) [reply dnq/src/main/kotlin/kotlinx/dnq/util/TransientEntityUtil.kt]: Oh, just to many merge requests made me a bit blind on changes i commit in little fixes. Sorry

**Review by andrii0lomakin — APPROVED** [review]

File: dnq/src/main/kotlin/kotlinx/dnq/util/TransientEntityUtil.kt (hunk 1 above)
Comment (andrii0lomakin) [reply dnq/src/main/kotlin/kotlinx/dnq/util/TransientEntityUtil.kt]: I do not believe a word :-)))

**Pull request merged**